.mergify.yml

```
@@ -10,3 +10,9 @@ pull_request_rules:
         assignees:
           - "{{ author }}"
         bot_account: "{{ author }}"
+defaults:
+    actions:
+        backport:
+            assignees:
+            - "{{ author }}"
+            bot_account: "{{ author }}"
```

The rest of this PR uses two spaces for indentation.  This is YAML, so I think spacing matters.  Either way, I think we should be consistent.   How about this, does this look right? 
```suggestion
defaults:
  actions:
    backport:
      assignees:
        - "{{ author }}"
      bot_account: "{{ author }}"
```

---

Weird, it all lines up in the online editor.